features/.settings/.gitignore

```diff
@@ -0,0 +1,3 @@
+/org.eclipse.core.resources.prefs
+/org.eclipse.jdt.core.prefs
+/org.eclipse.m2e.core.prefs
```

Comment:
I would remove this file and update the .gitignore located at the root folder of this repository to ignore .settings